src/Games/NexusMods.Games.AdvancedInstaller.UI/Resources/Language.pl.resx

```diff
@@ -43,7 +49,7 @@
 		<value>DUPLIKAT FOLDERU</value>
 	</data>
 	<data name="Pill_FOLDER_MERGED" xml:space="preserve">
-		<value>POŁĄCZONY FOLDER</value>
+		<value>FOLDER SCALONY</value>
```

Comment:
In the context of the Advanced Installer, the pill is shown when the result of 2 folders.

I don't mind going for 'scalony', but I'm not sure if it's user friendly. I'm not sure how common the word is in regular Polish use (I speak only with my family). It was certainly new for me as I had to look it up.

Reply:
`Scal, Scalanie` or `Scalony`.

**Check:**
[Scalony folder systemowy](https://answers.microsoft.com/pl-pl/windows/forum/all/scalony-folder-systemowy/e1929b6c-3557-4a48-9041-b7d599577d1e)
[Scal foldery i pliki w Azure Repos](https://learn.microsoft.com/pl-pl/azure/devops/repos/tfvc/merge-folders-files?view=azure-devops)
[Łączenie lub scalanie plików do postaci pojedynczego PDF](https://helpx.adobe.com/pl/acrobat/using/merging-files-single-pdf.html)

```
Combline = Połącz
Combline Files = Połącz pliki
```

`FOLDER MERGED` = `FOLDER SCALONY (the folder has already been merged)` or `SCALONY FOLDER (the folder is currently merged)`
`MERGE FOLDER` = `SCAL FOLDER (user clicks button when he wants to merge folder)`

`Połączony` is OK, but someone might confuse it with the word `Connected` or `Combine`.

I know that Poles use in Mod Managers (I think **Mod Organizer 2**, but I'm not 100% sure) `Scal` (`Merge`).

```
Merge = Scal
Replace = Zamień
Rename = Zmień nazwę
Cancel = Anuluj
```

![1](https://github.com/Nexus-Mods/NexusMods.App/assets/31535921/fc52eb6c-a5e3-4344-b0bd-229b8f8a697d)

What do you think?

Reply:
Currently it should fit in the UI. I will test the interface in the future and simply change the word if necessary.